drivers/gles2/shaders/cubemap_filter.glsl

```diff
@@ -40,8 +40,8 @@ void main() {
 #endif
 
 #if !defined(GL_EXT_shader_texture_lod) && !defined(GL_ARB_shader_texture_lod)
-#define texture2DLod(img, coord, lod) texture2D(img, coord, lod)
-#define textureCubeLod(img, coord, lod) textureCube(img, coord, lod)
+#define texture2DLod(img, coord, lod) texture2D(img, coord)
+#define textureCubeLod(img, coord, lod) textureCube(img, coord)
```

Comment:
Im surprised that this makes a difference as any call to texture2DLod or textureCubeLod has 0.0 as the lod offset.

Reply:
It doesn't make a difference in this case (even setting it to a large negative number didn't change anything, but if I understand it correctly, it should force sampling from mip 0.)

In this specific shader, we always want to sample from mip 0, as the other mips of the cubemap source texture aren't set (and are just black.)